src/internal.c

```diff
@@ -20679,7 +20734,27 @@ const char* GetCipherEncStr(char n[][MAX_SEGMENT_SZ]) {
 
     return encStr;
 }
-
+/* Check if a cipher is AEAD
+ * @param n return segment cipher name
+ * return 1 if the cipher is AEAD, otherwise 0
+ */
+int IsAEAD(char n[][MAX_SEGMENT_SZ])
```

Comment:
Consider checking "n" for NULL, in case other people call this local function in the future.

Reply:
That is correct. Added check that "n" is not NULL. Removed printf . 
Renamed function name because the name seemed generic.